core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -355,6 +356,10 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
     val cromwellSubmission = for {
       (wdl, workflowRecs, wfInputsBatch, wfOpts, wfLabels, wfCollection, dosUris, petUserInfo) <- workflowBatchFuture
       dosServiceAccounts <- resolveDosUriServiceAccounts(dosUris, petUserInfo)
+      // For Jade, HCA, anyone who doesn't use Bond, we won't get an SA back and the following line is a no-op
+      // We still call Martha for those because we can verify the user has permission on the DRS object as
+      // early as possible, rather than letting the workflow(s) launch and fail
+      // AEN 2020-09-08 [WA-325]
```

Comment:
If we follow my suggestion (in a subsequent PR), then this comment will no longer be true.  In order to not hammer our 3rd party partners, we might prefer to let the workflows launch and fail rather than fail fast up front since our partners cannot handle that load, it seems.